### PR TITLE
Adds DOMTokenList compat data

### PR DESCRIPTION
**Pull request by jafow**
Hello!
First time PR here, long time MDN docs fan. 
Adds the [DOMTokenList](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList) compat data.

### CODE REVIEW

File: api/DOMTokenList.json [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+            "version_added": true
+          },
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/DOMTokenList.json]: This should be `true`.

File: api/DOMTokenList.json [hunk 2]
```diff
@@ -0,0 +1,757 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "iterable_methods": {
+        "__compat": {
+          "description": "Iterable methods <code>entries()</code>,<code>keys()</code>, <code>forEach()</code>, <code>values()</code>",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DOMTokenList.json]: Should be `true`.

File: api/DOMTokenList.json [hunk 3]
```diff
@@ -0,0 +1,757 @@
...
+              "version_added": "50"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

File: api/DOMTokenList.json [hunk 4]
```diff
@@ -0,0 +1,757 @@
...
+              "version_added": "49"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

File: api/DOMTokenList.json [hunk 5]
```diff
@@ -0,0 +1,757 @@
...
+            },
+            "ie": {
+              "version_added": true,
+              "notes": " IE only trims whitespace — it doesn't remove duplicates."
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

File: api/DOMTokenList.json [hunk 6]
```diff
@@ -0,0 +1,757 @@
...
+              "version_added": true
+            },
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

File: api/DOMTokenList.json [hunk 7]
```diff
@@ -0,0 +1,757 @@
...
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

File: api/DOMTokenList.json [hunk 8]
```diff
@@ -0,0 +1,757 @@
...
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

File: api/DOMTokenList.json [hunk 9]
```diff
@@ -0,0 +1,757 @@
...
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

File: api/DOMTokenList.json [hunk 10]
```diff
@@ -0,0 +1,757 @@
...
+              "version_added": "50"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

File: api/DOMTokenList.json [hunk 11]
```diff
@@ -0,0 +1,757 @@
...
+            },
+            "edge": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

File: api/DOMTokenList.json [hunk 12]
```diff
@@ -0,0 +1,757 @@
...
+              "version_added": "50"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

File: api/DOMTokenList.json [hunk 13]
```diff
@@ -0,0 +1,757 @@
...
```
Comment (jpmedley) [inline api/DOMTokenList.json]: true

**Comment by jafow** [comment]
Hello @jpmedley thanks for reviewing. Sorry for so many mistakes on the opera mobile!

Could you point out how to verify opera mobile support, because I'd like to do more but without so many errors next time 😄 

For example, I noticed that on the current browser compat table for the `.add()` method [https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/add](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/add) the mobile tab shows Opera Mobile as `?`. 

It seems like Opera Mobile isn't the same as Opera android -- am I understanding that correctly? If so, I'll dig more next time to verify support of a feature for Opera android specifically. 

Thanks a lot!

File: api/DOMTokenList.json [hunk 14]
```diff
@@ -0,0 +1,757 @@
...
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "replace": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMTokenList/replace",
+          "description": "<code>replace()</code>",
```
Comment (Elchi3) [inline api/DOMTokenList.json]: The description is redundant here, please remove.

File: api/DOMTokenList.json [hunk 15]
```diff
@@ -0,0 +1,757 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "iterable_methods": {
```
Comment (Elchi3) [inline api/DOMTokenList.json]: The entire entry ""iterable_methods" is redundant with the entries for the various methods. Please remove.

File: api/DOMTokenList.json [hunk 16]
```diff
@@ -0,0 +1,757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "supports": {
+        "__compat": {
+          "description": "<code>supports()</code>",
```
Comment (Elchi3) [inline api/DOMTokenList.json]: Remove description. Not needed

File: api/DOMTokenList.json [hunk 17]
```diff
@@ -0,0 +1,757 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "remove_whitespace_and_duplicates": {
+        "__compat": {
+          "description": "Remove whitespace and duplicates",
```
Comment (Elchi3) [inline api/DOMTokenList.json]: I think "Trimming of whitespace and removal of duplicates" is a better description and matches the section in the doc.

File: api/DOMTokenList.json [hunk 18]
```diff
@@ -0,0 +1,757 @@
...
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": "11",
+              "notes": "IE9 and below - NO; Windows 10, IE 11.608 - Yes"
```
Comment (Elchi3) [inline api/DOMTokenList.json]: This note is not needed. Lets remove it.

**Comment by jafow** [comment]
Thanks a lot 👍; added those updates here.

**Comment by daniele-sforza** [comment]
Hi!
The compatibility table for [DOMTokenList.replace()](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/replace) method reports Edge is supported but it's not.
I had an error while debugging this in Edge and I found this issue as well https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/11297888/

> Major part of the Edge failures is due to a missing DOMTokenList.prototype.replace().

**Comment by Elchi3** [comment]
@jafow I see you did some commits that correct this. Want to open a PR as well? (or maybe I can't find it right now).

**Comment by jafow** [comment]
Hi @Elchi3 yes PR is [here](https://github.com/mdn/browser-compat-data/pull/1731)

**Pull request merged**